### ISSUE TITLE
[Feature]: add failsafe if labels in alt text/captions csv contain "fig-"

### ISSUE THREAD

**Issue by sbreitbart-NOAA** [Schiano-NOAA/ASAR#444]
### Is your feature request related to a problem? Please describe.

It's understandable that someone would set up the workflow with identical labels in the captions/alt text csv and code chunks. However, labels in the csv don't need to contain "fig-". If they do, add_alttext() will likely fail because the labels in the code chunks are identical to the labels in the csv. Labels in the csv need to be identical to labels in the code chunks MINUS "fig-" or "tab-".

### Describe the solution you would like.

Add failsafe that checks alt text/captions csv's "labels" column and alerts user if if labels contain "fig-" or "tab-". 

### Describe alternatives you have considered

NA

**Comment by sbreitbart-NOAA** [comment]
This may be moot if we end up altering the process for adding alt text to reports by using Quarto v1.9's chunk option fig-alt instead of add_alt_text()
